src/main/pg/gps.c

```diff
@@ -29,7 +29,7 @@
 
 #include "gps.h"
 
-PG_REGISTER_WITH_RESET_TEMPLATE(gpsConfig_t, gpsConfig, PG_GPS_CONFIG, 4);
+PG_REGISTER_WITH_RESET_TEMPLATE(gpsConfig_t, gpsConfig, PG_GPS_CONFIG, 5);
```

Comment:
We should not have increased the PG ID previously in #13240 since no new parameters were added to the PG in that PR.
We should not add it now since there is no need to default custom nmea commands to "".  `nmeaCustomCommands` are empty in all CLI dumps and diff all that I've seen.

Reply:
Agree. Restored PG version.